src/integrationTest/kotlin/com/sourcegraph/cody/edit/DocumentCodeTest.kt

```diff
@@ -21,93 +19,65 @@ import org.junit.runner.RunWith
 @RunWith(CustomJunitClassRunner::class)
 class DocumentCodeTest : CodyIntegrationTextFixture() {
 
-  @Test
   @Ignore
```

Comment:
so it still fails, right?

Reply:
I think the way it is written there is no way it could be stable, as there is race in the test itself.
I will try to rewrite it a bit.